extension-config.json

```diff
@@ -0,0 +1,37 @@
+{
+  "version": "0.1.0",
+  "id": "@shopgate/pdp-native-share",
+  "components": [
+    {
+      "id": "ShareButton",
+      "path": "frontend/portals/ShareButton",
+      "target": "product.ctas.before",
+      "type": "portals"
+    }
+  ],
+  "steps": [
+  ],
+  "configuration": {
+    "iOSShareSVG": {
+      "type": "admin",
+      "destination": "frontend",
+      "default": "M9.26855 4.65566L9.64551 5.05801C9.7373 5.15371 9.88379 5.15566 9.97363 5.06289L13 1.99892V17.0043H14.001V1.87051L17.2529 5.12246C17.3418 5.21377 17.4873 5.21182 17.5781 5.11709L17.9922 4.72158C18.084 4.62685 18.0859 4.47549 17.9961 4.38369L14.2305 0.617089C14.2236 0.565819 14.2002 0.515526 14.1621 0.475487L13.7852 0.0731434C13.6924 -0.0225598 13.5459 -0.0245129 13.457 0.0682606L9.2627 4.31435C9.17285 4.40713 9.17578 4.56045 9.26855 4.65566ZM5 25.5043V8.5043H11V7.5043H4V26.5043H23V7.5043H16V8.5043H22V25.5043H5Z",
```

Comment:
In my opinion this approach won't really work in real life.

It's not so easy to just replace an SVG path with another and assume it would look correct.
Also, even if this is not a problem, it would only cover one case -> merchant want to exchange one svg path with another. If required icon has more paths, or ellipsis, or whatever more which svg supports, this won't be covered.

I talked to our PO and what she really want to cover right now is merchants who wants to have iOS icon on GMD and vice versa.

I suggest to do it other way:
1. Provide and extension config that would be something like:
iosIconStyle: 'gmd' or 'ios',
gmdIconStyle: 'gmd' or 'ios'
Depending on the configuration, extensions renders this or that.

2. Provide a portal position that would give  the possibility to exchange the icon completely.

Reply:
Changed to icon style by string. Not sure if string is best method to use